src/quickpicks/schemaRegistries.ts

```diff
@@ -142,7 +138,7 @@ function populateLocalSchemaRegistries(
   });
 
   // Will most likely be a single local Schema Registry, but the API is designed to support multiple.
-  localSchemaRegistries.forEach((schemaRegistry: LocalSchemaRegistry) => {
+  localSchemaRegistries.forEach((schemaRegistry: SchemaRegistry) => {
```

Comment:
Same question as above

Reply:
Fixed.

Reply:
Looks like this is still `SchemaRegistry`?